### DISCUSSION TITLE
freepik.com

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1418] [Personal Filter]
freepik.com

![image](https://github.com/realodix/AdBlockID/assets/19517680/1f9b2b5e-ce21-45ca-b7ee-61c5f4645e62)

https://www.freepik.com/search?format=search&page=2&query=baliho+idul+adha&type=vector

tolong bikinkan filter block popup tsb, atau klo memungkinkan dimasukkan ke adblockid plus, popupnya mengganggu banget selalu muncul

**Comment by realodix** [comment]
Halo, hingga hari ini Saya belum menemukan solusinya. Mungkin ada seseorang di luar sana yang bisa membantu menemukan solusinya.
